package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "remove",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/react-router": "^4.0.15",
+    "@types/react-router-dom": "^4.0.8",
```

Comment:
@types must be in dev-dependencies.

Reply:
Done in the branch Mathieu/feat/LoginPage